doc/ref/states/backup_mode.rst

```diff
@@ -24,6 +24,14 @@ Or it can be set for each file:
         - source: salt://ssh/sshd_config
         - backup: minion
 
+
+The backup_mode can be set to any of thefollowing settings:
```

Comment:
**Nitpick (blocking, typo):** Should be "the following" instead. 

**Suggestion (non-blocking, phrasing):** I'd suggest possibly using "options "instead of "settings" for clarity.